engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/SchemaRegistryClient.scala

```diff
@@ -1,9 +1,11 @@
 package pl.touk.nussknacker.engine.avro.schemaregistry
 
+import java.io.Closeable
+
 import cats.data.Validated
 import org.apache.avro.Schema
 
-trait SchemaRegistryClient extends Serializable {
+trait SchemaRegistryClient extends Serializable with Closeable {
```

Comment:
Where do we close it in our code? E.g. in SourceFactory, etc. ?

Reply:
've removed all close methods. GC should clean up all.